best-practices/README.md

```diff
@@ -36,102 +29,40 @@ Ruby
 ----
 
 * Avoid optional parameters. Does the method do too much?
-* Avoid monkey-patching.
-* Generate necessary [Bundler binstubs] for the project, such as `rake` and
-  `rspec`, and add them to version control.
-* Prefer classes to modules when designing functionality that is shared by
-  multiple models.
-* Prefer `private` when indicating scope. Use `protected` only with comparison
-  methods like `def ==(other)`, `def <(other)`, and `def >(other)`.
 
 [Bundler binstubs]: https://github.com/sstephenson/rbenv/wiki/Understanding-binstubs
 
 Ruby Gems
 ---------
 
-* Declare dependencies in the `<PROJECT_NAME>.gemspec` file.
-* Reference the `gemspec` in the `Gemfile`.
-* Use [Appraisal] to test the gem against multiple versions of gem dependencies
-  (such as Rails in a Rails engine).
 * Use [Bundler] to manage the gem's dependencies.
-* Use [Travis CI] for Continuous Integration, indicators showing whether GitHub
-  pull requests can be merged, and to test against multiple Ruby versions.
 
-[Appraisal]: https://github.com/thoughtbot/appraisal
 [Bundler]: http://bundler.io
-[Travis CI]: http://travis-ci.org
 
 Rails
 -----
 
 * Avoid bypassing validations with methods like `save(validate: false)`,
   `update_attribute`, and `toggle`.
-* Avoid instantiating more than one object in controllers.
-* Avoid naming methods after database columns in the same class.
 * Don't change a migration after it has been merged into master if the desired
   change can be solved with another migration.
 * Don't reference a model class directly from a view.
-* Don't return false from `ActiveModel` callbacks, but instead raise an
-  exception.
 * Don't use instance variables in partials. Pass local variables to partials
   from view templates.
-* Don't use SQL or SQL fragments (`where('inviter_id IS NOT NULL')`) outside of
-  models.
-* Generate necessary [Spring binstubs] for the project, such as `rake` and
-  `rspec`, and add them to version control.
 * If there are default values, set them in migrations.
 * Keep `db/schema.rb` or `db/development_structure.sql` under version control.
-* Use only one instance variable in each view.
```

Comment:
Yep. I don't think we're there yet. Para tenerlo en mente en el futuro! O crees que sea buena idea meterlo de una?